index.html

```diff
@@ -7,15 +7,15 @@
   <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet"/>
   <link href="https://fonts.googleapis.com/css?family=Roboto+Mono" rel="stylesheet"/>
   <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet"/>
-  <link href="/static/app.css" rel="stylesheet"/>
-  <link href="/static/favicon.ico" rel="icon" type="image/x-icon"/>
+  <link href="/app.css" rel="stylesheet"/>
+  <link href="/favicon.ico" rel="icon" type="image/x-icon"/>
 </head>
```

Comment:
@bradrydzewski Do you have an idea, why on the dashboard there are no projects listed, even though the `window.STATE_FROM_SERVER` is filled properly?